pkg/controller/networkpolicy/crd_utils.go

```diff
@@ -155,3 +196,19 @@ func (n *NetworkPolicyController) getTierPriority(tier string) int32 {
 	}
 	return t.Spec.Priority
 }
+
+// isGroupSelectorUnset determines whether an internal Group corresponding to
+// a ClusterGroup is still pending create. This may occur because a ClusterGroup
+// might be referenced before it is actually created. In such a scenario, the
+// corresponding AddressGroup/AppliedToGroup will be created with empty Selectors
+// until the actual ClusterGroup create triggers the population of the remaining
+// fields.
+func isGroupSelectorUnset(g antreatypes.GroupSelector) bool {
```

Comment:
Should we change `Selector` in `antreatypes.Group` to be a pointer so we check rely on nil check here? With ipBlocks and serviceSelectors there's a good chance that group.Selector should be empty, so pointer might make sense here.

Reply:
this check is no longer need in this PR